bindings/wysiwyg-ffi/src/ffi_composer_model.rs

```diff
@@ -282,19 +271,11 @@ impl ComposerModel {
         self: &Arc<Self>,
         url: String,
         text: String,
-        attributes: Vec<Attribute>,
+        _attributes: Vec<Attribute>, // TODO remove attributes
     ) -> Arc<ComposerUpdate> {
         let url = Utf16String::from_str(&url);
         let text = Utf16String::from_str(&text);
-        let attrs = attributes
-            .iter()
-            .map(|attr| {
-                (
-                    Utf16String::from_str(&attr.key),
-                    Utf16String::from_str(&attr.value),
-                )
-            })
-            .collect();
+        let attrs = vec![];
```

Comment:
While this is a TODO to be removed, I'd prefer keep passing the `attributes` argument through to the composer. That way it still works as expected until it's removed.

Also, should we link an issue?

Reply:
Yep I think that's a fair call. 
Reverted in: [2ed5d77](https://github.com/matrix-org/matrix-rich-text-editor/pull/720/commits/2ed5d77d54c9d9d65a6eef00e31c86e5f887aecf)
Issue: https://github.com/matrix-org/matrix-rich-text-editor/issues/721